packages/vue/package.json

```diff
@@ -35,6 +35,9 @@
     "prettier": "1.16.4",
     "vue-template-compiler": "2.5.21"
   },
+  "publishConfig": {
```

Comment:
Right, for scoped packages, I wonder why this wasn't included in our initial vue PRs

Reply:
I didn't notice it, and the vue cli didn't generate it 😕 in the future It'd be great to have some tooling take care of that kind of project syncing